src/Microsoft.Graph.Core/Requests/BaseFunctionMethodRequestBuilder.cs

```diff
@@ -121,5 +116,17 @@ private string EscapeStringValue(string value)
             // Per OData spec, single quotes within a string must be escaped with a second single quote.
             return value.Replace("'", "''");
         }
+
+        /// <summary>
+        /// Set the parameter string on the request URL. MUST be used after all 
+        /// parameters have been set in the generated request builder. Used for
+        /// each section. Adds an empty () if no parameters are present.
+        /// </summary>
+        protected void SetParameterString()
+        { 
+            this.RequestUrl = string.Format("{0}({1})", 
+                                            this.RequestUrl, 
```

Comment:
What if `this.RequestURL` is null or  empty?

Reply:
Good question.

Composable OData functions are bound to a type. This means we'd have to have a request builder for the bound type that would call this function request builder. The request builder for the bound type would have a request URL that would be passed into the cstor of this type. If `this.RequestURL` is null or empty, we'd have problems elsewhere.